### PR TITLE
Added sounds example to docs

### PR DESCRIPTION
**Pull request by kolibril13**
Follow up to #926 as that pr got messed up while reverting commits.
This pr adds a sound example to the docs.

EDIT:
You can see the rendered example here:
https://manimce--1188.org.readthedocs.build/en/1188/reference/manim.scene.scene.Scene.html?highlight=click#soundexample

### CODE REVIEW

**Comment by WampyCakes** [comment]
Is there a way to start the video paused? Someone looking at Scene documentation probably does not want to immediately start hearing clicks for some reason unknown to them.

**Comment by kolibril13** [comment]
On a Mac (tested in chrome+safari), it does not autostart.
Maybe there is a way to autostart videos in the docs when they occur in the visible field?
Or we start the videos with volume 0 by default?

**Comment by behackl** [comment]
> On a Mac (tested in chrome+safari), it does not autostart.

From [the Safari documentation](https://developer.apple.com/documentation/webkit/delivering_video_content_for_safari):

> By default, autoplay executes only if the video doesn’t contain an audio track, or if the video element includes the `muted` attribute. Video playback pauses if the element gains an audio track, becomes unmuted without user interaction, or if the video is no longer onscreen (either by CSS or due to the user scrolling the page).

I'm pretty sure this isn't handled consistently across browsers.

Anyhow, `autoplay` usually only starts playing once the video enters the viewport. We could auto-mute the video by default, but that would require the introduction of further customization options in `manim_directive.py`: https://github.com/ManimCommunity/manim/blob/6ef0733a1ec38813bb5b71c2b93bb15e8e7acd16/docs/source/manim_directive.py#L295

Personally, I'm fine with this PR in its current form.

**Review by behackl — APPROVED** [review]

File: manim/scene/scene.py [hunk 1]
```diff
@@ -980,18 +980,31 @@ def update_to_time(self, t):
...
-        sound_file : str
-            The path to the sound file.
-
-        time_offset : int,float, optional
             The offset in the sound file after which
             the sound can be played.
 
-        gain :
+        gain : float
+            Amplification of the sound.
```
Comment (naveen521kk) [inline manim/scene/scene.py:987]: There is something off in the diff here, please check whether things look correct.

Comment (kolibril13) [reply manim/scene/scene.py]: thanks, I updated this

**Pull request merged**